src/lib/stitching/vortex/stitching.ts

```diff
@@ -16,6 +16,7 @@ const getMaxPrice = (thing: { listPrice: any }) => {
 export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
   // The SDL used to declare how to stitch an object
   extensionSchema: gql`
+    union AnalyticsEntityUnion = Artwork | Show | Artist
```

Comment:
Maybe we can call it `AnalyticsRankedEntityUnion` to be explicit?

Reply:
And... this is the magical part! 🎩

Reply:
👍 fixed in https://github.com/artsy/metaphysics/pull/1770/commits/ea759d1d59c8d27f800f88e3df7605a4051e374e